doc/security/specs/device_provisioning/index.md

```diff
@@ -0,0 +1,977 @@
+Unlock operations do not require CPU intervention. A test exit token is used to
+gate the device transition from TEST state into one of the final operational
+states (`DEV`, `PROD_END`, `PROD`).
+
+### Pre-Personalization
+
+The following steps illustrate the set of operations required prior to
+personalization.
+
+#### Wafer Test - Electrical Die Sorting (EDS)
+
+*   `RAW_UNLOCK`: Unlock TEST mode by providing unlock token via TAP interface.
+*   `ANALOG TRIM & TEST (EDS)`: Regular analog test, scan, calibration, trimming
+    and functional testing (EDS stage).
+*   `SET DEVICE ID`: Program the device identifier fuses. DEVICE ID export may
+    be required depending on the provisioning flow.
+*   `LOCK DEVICE`: (Optional) Program the TEST_UNLOCK (optional) and/or
+    TEST_EXIT tokens depending on the test flow configuration.
+
+    Optional: If required by the manufacturing flow, lock the device for safe
+    transport into the next test stage.
+
+#### Package TEst (TEST MODE) - Automated Test Equipment (ATE)
+
+*   `TEST_UNLOCK`: Unlock TEST mode by providing TEST_UNLOCK token via TAP
+    interface.
+*   `ANALOG TRIM & TEST (OSAT)`: Regular analog test, scan, calibration,
+    trimming and functional testing (EDS stage).
+*   `LOG DEVICE ID`: Record DEVICE_ID and test results.
+*   `LOCK DEVICE`: Program the TEST_UNLOCK (optional) and/or TEST_EXIT tokens
+    depending on test configuration.
+
+    Optional: If required by the manufacturing flow, lock the device for safe
+    transport into the next test stage.
+
+#### Package Test (DEV or PROD MODE) - Automated Test Equipment (ATE)
+
+*   `TEST_EXIT`: Unlock device and transition from TEST to one of the following
+    operational states: DEV, PROD_END, or PROD.
+*   `SKU SETTINGS`: SKU dependent fuse and info flash configuration.
+*   `PERSONALIZATION`: See [Personalization](#personalization) section for more
+*   `LOAD FW IMAGE`: Load factory image payload.
+
+## Personalization {#personalization}
+
+### Test Setup
+
+The following diagram shows the physical connections between test components.
+
+<table>
+  <tr>
+     <td>
+        <img src="img_ate_setup.svg" style="width: 800px;">
+     </td>
+  </tr>
+  <tr><td style="text-align:center;">Figure: FT Connectivity</td></tr>
+</table>
+
+#### Components
+
+*   **Device Under Test**: An OpenTitan device being tested as part of its
+    manufacturing process.
+*   **ATE**: Automatic Test Equipment (ATE), used to perform tests on the
+    devices as part of the manufacturing process.
+*   **Provisioning Appliance**: A network connected local server with an
+    attached HSM. The server implements an OpenTitan compliant secure boot
+    implementation, and runs signed applications used to communicate with ATE
+    and cloud services.
+*   **Provisioning Service**: Cloud service used to authenticate and initialize
+    provisioning appliances.
+
+    The service uses a Key Manager Service (KMS) to manage signing and
```

Comment:
Yes, I agree that this is not clear. I have removed this sentence as it does not provide additional details to the implementer. 

This can be covered in a separate document focused on the protocols between the provisioning service and the provisioning appliance.